### PR TITLE
Update routes

### PR DESCRIPTION
**Pull request by richa15997**
#7 This pull request contains the following:
1. Added app routes with an additional backslash to create and list record functions
2. Updated the index to show meaningful introductory results when hitting our flask application

### CODE REVIEW

**Review by aayushagrawal135 — COMMENTED** [review]
We may need slash handling read, update and delete as well.

**Review by SAR2652 — COMMENTED** [review]
Why not create a string variable called ```base_url = "/inventory-records"``` and then add the forward "/" for other routes using base_url as a variable?

File: service/routes.py [hunk 1]
```diff
@@ -55,6 +60,7 @@ def get_inventory_records(product_id):
     app.logger.info("Returning product: %s", product.name)
     return jsonify(product.serialize()),status.HTTP_200_OK
 
+@app.route("/inventory-records/", methods=["POST"])
```
Comment (sakshambassi) [inline service/routes.py:63]:
```
app = Flask('my_app')
app.url_map.strict_slashes = False```
Instead, can we do this? https://stackoverflow.com/questions/40365390/trailing-slash-in-flask-route

Comment (richa15997) [reply service/routes.py:63]: Good suggestion. I will follow this

**Pull request closed without merging**